google/cloud/storage/benchmarks/throughput_experiment.cc

```diff
@@ -369,23 +367,24 @@ std::vector<std::unique_ptr<ThroughputExperiment>> CreateUploadExperiments(
 
 std::vector<std::unique_ptr<ThroughputExperiment>> CreateDownloadExperiments(
     ThroughputOptions const& options,
+    google::cloud::storage::Client rest_client,
+    google::cloud::storage::Client grpc_client,
     google::cloud::Options const& client_options, int thread_id) {
-  gcs::Client rest_client(client_options);
+  if (options.client_per_thread) rest_client = gcs::Client(client_options);
+#if GOOGLE_CLOUD_CPP_STORAGE_HAVE_GRPC
+  if (options.client_per_thread) {
+    grpc_client = DefaultGrpcClient(client_options);
+  }
+#else
```

Comment:
A perhaps-naive question: Why is this different from `CreateUploadExperiments()`?

Reply:
Not naive at all, fixed.